egs/yesno/ASR/local/compute_fbank_yesno.py

```diff
@@ -42,7 +42,9 @@ def compute_fbank_yesno():
     )
     assert manifests is not None
 
-    extractor = Fbank(FbankConfig(num_mel_bins=num_mel_bins))
+    extractor = Fbank(
+        FbankConfig(sampling_rate=8000, num_mel_bins=num_mel_bins)
```

Comment:
Is the purpose of specifying the sampling rate to pre-compute some constants during the constructor?

Please also update the fbank config used in the on the fly feature extractor in asr_datamodule.py in yesno.

Reply:
Yeah, it precomputes the mel bands. I think the same thing is being done in kaldifeat.

Will update, thanks.